drivers/nrf9160_gps/Kconfig

```diff
@@ -27,8 +27,7 @@ config NRF9160_GPS_FIX_CHECK_INTERVAL
 config NRF9160_GPS_HANDLE_MODEM_CONFIGURATION
 	bool "The GPS driver will handle modem configuration directly"
 	default y
-	select AT_CMD
-	select AT_CMD_PARSER
+	depends on !LWM2M_CARRIER && AT_CMD && AT_CMD_PARSER
```

Comment:
I made this change to get out of a dizzying dependency loop when depending on `!LWM2M_CARRIER`. Let me know if there are better ways to do it.